client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab/usePersonalInfoTabInterfaces.ts

```diff
@@ -0,0 +1,22 @@
+import { UseFormMethods } from 'react-hook-form';
+
+import SubOccupationAndStreet from 'models/SubOccupationAndStreet';
+import investigatedPatientRole from 'models/investigatedPatientRole';
+import { PersonalInfoDbData } from 'models/Contexts/PersonalInfoContextData';
+
+import { PersonalInfoTabState } from '../PersonalInfoTabInterfaces';
+
+export interface usePersonalInfoTabIncome {
+
+}
+
+export interface usePersonalInfoTabOutcome {
```

Comment:
why is outcome necessary? I mean Typescript can already infer what's the return type of the objects you return

Reply:
yes in my opinion, typing is always recommended in my opinion